### PR TITLE
docs: updated the documentation for #1880

### PR DESCRIPTION
**Pull request by mssola**
The pull request #1880 introduced two new environment variables. This
commit updates the documentation explaining both of them.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _docs/deploy.md [hunk 1]
```diff
@@ -67,6 +67,17 @@ be aware of the following requirements:
...
     restart Portus.
+- *Advanced*: the [official Docker
+  image](https://hub.docker.com/r/opensuse/portus/) assumes that you only want
+  to expose the Puma process externally (the actual process running
+  Portus). However, in some unusual deployments (such as the one described
+  [here](https://flavio.castelli.me/2018/07/18/hackweek-project-docker-registry-mirror/)),
+  you may want to expose this process only in a Unix socket (i.e. in short,
+  because you are sharing this socket with other processes). If you want Portus
+  to do this, you have to set the `PORTUS_PUMA_USE_UNIX_SOCKET` environment
+  variable to true. Note that the default behavior should be fine for the vast
```
Comment (flavio) [inline _docs/deploy.md]: to `"true"`

**Comment by flavio** [comment]
Minor glitch to be fixed, the remaining is fine.

File: _docs/deploy.md (hunk 1 above)
Comment (mssola) [reply _docs/deploy.md]: Done

**Comment by mssola** [comment]
I've already pushed into `gh-pages`, so the documentation should be updated shortly.

**Pull request merged**